### PR TITLE
Bump Protocol to 3bc1349b64249794d8e5cef8ac1ae368b73bacab

### PR DESCRIPTION
**Pull request by bundabrg**
Needs some tests but will fix https://github.com/bundabrg/GeyserReversion/issues/17

### CODE REVIEW

File: connector/pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>com.github.CloudburstMC.Protocol</groupId>
             <artifactId>bedrock-v419</artifactId>
-            <version>ce59d39118</version>
+            <version>3bc1349b64249794d8e5cef8ac1ae368b73bacab</version>
```
Comment (rtm516) [inline connector/pom.xml:35]: Change this to the short commit hash.
```suggestion
            <version>3bc1349</version>
```

**Comment by Hellohi3654** [comment]
Causes this error
image

**Comment by rtm516** [comment]
> Causes this error
> image

To fix this can we test and bump this PR to a newer version since it was fixed in the latest version (https://github.com/CloudburstMC/Protocol/commit/91161b5d193a71a82654b2957d3fbc74b70d8745)

**Comment by Hellohi3654** [comment]
so if a change 
<version>3bc1349b</version>
to
<version>91161b5</version>
it should work?

**Comment by DoctorMacc** [comment]
Now irrelevant.

**Pull request closed without merging**